code/obj/item/organ_holder.dm

```diff
@@ -826,6 +826,14 @@
 						H.wear_mask = newHead.wear_mask
 						newHead.wear_mask.set_loc(H)
 						newHead.wear_mask = null
+					if (isskeleton(H))
+						var/datum/mutantrace/skeleton/S = H.mutantrace
+						if (S.head_tracker != null)
+							S.head_tracker.linked_human = null
+						newHead.head_type = HEAD_SKELETON
+						newHead.linked_human = H
+						H.eye = H
+						S.set_head(newHead)
```

Comment:
Should only `set_head` if it's a skeleton head - you shouldn't be able to use non-skeleton heads like skeleton heads and see/talk/listen through them. Suggest just putting a `HEAD_SKELETON` check above on line 829.